exercise/map.js

```diff
@@ -0,0 +1,56 @@
+/*
+## Map
+
+### Instructions
+
+Create an array named 'arr2' that contains the values of 'arr1'
+with the following modifications applied to each of the values:
+1) They have to be multiplied by 2
+2) Then the results have to be subtracted by 4
+3) finally, the negative values have to be replaced by null
+then do the same with arrRandom1 by returning arrRandom2
+The use of `for` and `while` is forbidden for this exercise
+
+### Notions
+
+- Map
+
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
+
+*/
+let arr1 = [10, -10, 20, -95, 86]
```

Comment:
this array is so small they can just write the solution doing the math in theyr head.
You don't even need JS for that haha

Reply:
I wanted to show an easy "example" one along with a random one.